### ISSUE TITLE
slate-liquid-asset-loader should handle assets_url's with additional filters

### ISSUE THREAD

**Issue by zomars** [Shopify/slate#558]
### Problem

I have defined icons for each collection in my assets folder, I'm trying to retrieve them via handle.

This works: 
```liquid
{% capture collection_icon %}{{ '../assets/images/icon-abarrotes.png' | asset_url }}{% endcapture %}
<img src="{{ collection_icon }}" alt="">
```

This doesn't:
```liquid
{% capture collection_icon %}{{ '../assets/images/icon-%HANDLE%.png' | replace: '%HANDLE%', collection.handle | asset_url }}{% endcapture %}
<img src="{{ collection_icon }}" alt="">
```
This doesn't either:
```liquid
{% assign collection_icon_url = '../assets/images/icon-%HANDLE%.png' | replace: '%HANDLE%', collection.handle %}
{% capture collection_icon %}{{ collection_icon_url | asset_url }}{% endcapture %}
<img src="{{ collection_icon }}" alt="">
```

Maybe I'm missing something from the docs?

**Comment by zomars** [comment]
I see, thanks for the response. TBH I found very confusing which assets are handled locally and which don't. I'm still wrapping my head around this workflow. Thank you again!

**Comment by zomars** [comment]
And sorry, yes I've tried using said command and the most iI could get was a relative URL. 
```
<img src="../assets/images/icon-abarrotes.png" alt="">
```

**Comment by t-kelly** [comment]
Assets have been simplified in #850 and the change is now available in v1.0.0-beta.12.

Docs that have also been updated:

https://github.com/Shopify/slate/wiki/Slate-themes
https://github.com/Shopify/slate/wiki/Template-and-layout-bundles
https://github.com/Shopify/slate/wiki/Local-development
